Inclet/Ironji_mobile_api
# verify phone verification code Feature

#### What does this PR do?
This PR Verify code send by a user. **Verification code must be 4 digits**
#### Description of Task to be completed?
- Created controller for verifying the code sent by a user. 
- added tests for this feature
#### How should this be manually tested?
- after cloning this repo from https://github.com/Inclet/Ironji_mobile_api.git, check out to develop by typing `git checkout develop` in you r terminal.
- To install all dependencies run `npm install`
- To start server run `npm start` in your terminal.
- send you `verificationCode` through the request body of `/api/user/verify_phone_verification_code`  with a `PUT http method.`
#### Any background context you want to provide?
- You need to first initiate the usertype by passing `userType ` through `api/user/initiate_user_type` request body with a `POST HTTP method`,
- copy the token from the response body and use it in other request's headers.
- After, you need to send a verification code using `api/user/send_phone_verification_code` and pass in your `phoneNumber` through the request body.

Sending requests to the server you may use **postman (a Chrome browser extension)**
#### Screenshots (if appropriate)
<img width="1134" alt="Screen Shot 2019-06-02 at 13 55 39" src="https://user-images.githubusercontent.com/37753967/58760877-28c55b80-853e-11e9-8883-ba0c7e87d997.png">
